test/pressurehandler.test.js

```diff
@@ -18,92 +19,136 @@ function block (msec) {
 }
 
 test('health check', async t => {
-  t.plan(3)
-  const fastify = Fastify()
+  test('simple', async t => {
```

Comment:
I took the chance to nest health check related tests under a same parent test. There are no functional changes outside of the new test I added though, except the removal of some unnecessary t.plan

Reply:
I have found that it is clearer to have a `t.plan` when 1. the test function is an `async` function and 2. assertions are inside of a callback within the test. If the callback is never invoked, and the `t.plan` is not present, tap will recognize the non-thrown promise as a success and the test will pass. With the plan defined, regardless of the state of the promise tap will recognize that not all assertions were passed and fail the test.

Reply:
yes, those are the only t.plan I left. otherwise, with the availability of async tests in tap, t.plan is no longer necessary except in those specific cases where assertions are made in callbacks